src/modules/games/tictactoe/components/StartScreen/StartScreen.tsx

```diff
@@ -0,0 +1,50 @@
+import { useState } from 'react'
+
+type Props = {
+  onStart: (players: { p1: string; p2: string }) => void
+}
+
+export default function StartScreen({ onStart }: Props) {
+  const [player1, setPlayer1] = useState('')
+  const [player2, setPlayer2] = useState('')
+
+  const handleStart = () => {
+    if (!player1.trim() || !player2.trim()) return
```

Comment:
Boa logica, porem quando o texto for persistido na store esse trim() pode ir na hora que a store receber o dado